setup-toolchain.sh

```diff
@@ -5,9 +5,7 @@ set -e
 
 cd "$(dirname "$0")"
 
-if [[ "$CI" == true ]] || ! command -v rustup-toolchain-install-master > /dev/null; then
-    cargo install -Z install-upgrade rustup-toolchain-install-master --bin rustup-toolchain-install-master
-fi
+cargo install -Z install-upgrade rustup-toolchain-install-master --bin rustup-toolchain-install-master
```

Comment:
```suggestion
if [[ -x $HOME/.cargo/bin/rustup-toolchain-install-master ]]; then
    cargo install -Z install-upgrade rustup-toolchain-install-master
fi
```
This way RTIM on .cargo/bin will always up-to-date, and people install RTIM on other locations will not affected.

Reply:
or we could just check for ~/cargo/bin
```suggestion
if [[ -x ~/.cargo/bin/rustup-toolchain-install-master]]; then
```